api/src/clojure/org/akvo/flow_api/datastore/survey.clj

```diff
@@ -20,11 +20,11 @@
            (str (ds/id survey))))))
 
 (defn cached-list-ids [{:keys [survey-list-cache] :as remote-api} instance user-id]
-  (if-let [survey-list (cache/lookup @survey-list-cache [instance user-id])]
+  (if-let [survey-list (cache/lookup @(:cache survey-list-cache) [instance user-id])]
```

Comment:
same previous comment applies to this change 🤔

Reply:
For the new check_perms cache we are reusing a "cache" component (defined at https://github.com/akvo/akvo-flow-api/blob/46c3f24dff26cbcb12e3231e62840ff9eed879b3/api/src/clojure/org/akvo/flow_api/component/cache.clj#L8) which we did not notice that contains the actual cache object in the :cache slot